### PR TITLE
Extend async tasks to all GET endpoints

### PR DESCRIPTION
**Pull request by didi955**

### CODE REVIEW

**Comment by didi955** [comment]
I've extended the async methods to all GET requests in the API and removed unnecessary imports.

**Review by RaphHuynh — APPROVED** [review]
Merci pour l'envoie, j'allais le faire aujourd'hui mais du coup tu me fais gagner du temps.

File: app/main.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
-from typing import List, Annotated
+from typing import List
 
 from starlette.middleware.cors import CORSMiddleware
 
 from app.lib.sql import *
 
-from fastapi import FastAPI, Response, File, UploadFile
-from app.models import MemberIn, MemberOut, Category, CategoryOut, MemberWithCategory, MemberHasCategoryIn, GetMemberHasNetwork, MemberById, Network, MemberHasNetwork, MemberHasCategory, MemberHasNetworkIn
```
Comment (kantandane) [inline app/main.py:8]: Why not using imports with parentheses ? 
```py
from app.models import (
    MemberIn,
    MemberOut,
    Category,
    CategoryOut,
    MemberWithCategory,
    MemberHasCategoryIn,
    GetMemberHasNetwork,
    MemberById,
    Network,
    MemberHasNetwork,
    MemberHasCategory,
    MemberHasNetworkIn
)
```

or even better, 
```py
from app import models
```
and in the code, for example 

```py
@app.get("/members", response_model=List[models.MemberWithCategory])
async def api_get_members():
    return await get_members()

```

File: app/lib/sql.py [hunk 2]
```diff
@@ -1,8 +1,7 @@
 import asyncio
 from collections import namedtuple
-from typing import Annotated
 
-from fastapi import File, UploadFile
+from fastapi import UploadFile
 
 from app.models import GetMembers, MemberIn, Category, CategoryOut, MemberWithCategory, MemberHasCategoryIn, \
```
Comment (kantandane) [inline app/lib/sql.py:6]: Why not using imports with parentheses ? 
```py
from app.models import (
    MemberIn,
    MemberOut,
    Category,
    CategoryOut,
    MemberWithCategory,
    MemberHasCategoryIn,
    GetMemberHasNetwork,
    MemberById,
    Network,
    MemberHasNetwork,
    MemberHasCategory,
    MemberHasNetworkIn
)
```

or even better, 
```py
from app import models
```
and in the code, for example 

```py
@app.get("/members", response_model=List[models.MemberWithCategory])
async def api_get_members():
    return await get_members()

```

**Pull request merged**